assets/js/components/Accordion.js

```diff
@@ -44,12 +45,26 @@ export default function Accordion( {
 		}
 	}, [ isActive, onClose, onOpen ] );
 
+	useEffect( () => {
+		if ( disabled && isActive ) {
+			setActive( false );
+		}
+	}, [ disabled, isActive ] );
```

Comment:
Ah good point. This isn't great because it creates a layout shift where the item under the mouse changes.

My first thought is that the behavior of the accordion only toggles the visibility of the description which doesn't necessarily need to be disabled by the selection limits (only the checkboxes really need to be disabled). It could be useful to be able to see what the descriptions are without deselecting one to be able to read it.

Let's restore the previous behavior to avoid the layout shift and address this in a follow-up as we probably need to consult with UX.

Reply:
I agree. The accordion doesn't need to be disabled in any case. I've restored the previous behavior and will open a follow-up issue to discuss about the accordion. Thanks!

Reply:
@nfmohit Do we have a new issue for the follow-up?

Reply:
Just opened #7383 for the follow-up. Apologies for the delay!